src/main/java/digital/slovensko/autogram/ui/gui/GUIApp.java

```diff
@@ -48,6 +50,7 @@ public void start(Stage windowStage) throws Exception {
                     var thread = new Thread(server::stop);
                     windowStage.setOnCloseRequest(event -> {
                         thread.start();
+                        finalAutogram.stopTokenSessionTimer();
```

Comment:
toto mi pride, ze ukazujeme moc vnutornosti, nezabalime to do nejakeho `autogram.shutdown` alebo nieco take? Nemame uz nieco take vlastne pre server?

Reply:
server existuje nad Autogramom, nie v ňom. Ale hej, pridal som Autgoram::shutdown